eval/src/vespa/eval/eval/test/tensor_conformance.cpp

```diff
@@ -1267,7 +1267,7 @@ struct TestContext {
         MappedFileInput file(path);
         Slime slime;
         EXPECT_TRUE(file.valid());
-        EXPECT_EQUAL(JsonFormat::decode(file, slime), file.get().size);
+        EXPECT_TRUE(JsonFormat::decode(file, slime));
```

Comment:
Same as above

Reply:
The original idea behind the API is that the user decides if he cares about ok/fail or the actual size. We could change the API itself to return bool now that we have a version taking Input that will track data consumption. bool makes sense for json format since how much of the input we actually consume depends on whitespace (typically trailing \n)

Reply:
I don't think we need to change the API. It should be sufficient to change the tests that actually care about the size to be more explicit (e.g. used > 0).